app/src/main/java/com/simplecity/amp_library/ui/screens/album/menu/AlbumMenuPresenter.kt

```diff
@@ -129,8 +135,13 @@ class AlbumMenuPresenter @Inject constructor(
         const val TAG = "AlbumMenuContract"
     }
 
-    fun newUiEvent(uiEventType: UiEventType, album: Album){
-            val uiEvent = EventUtils.newUiAlbumEvent(album, uiEventType)
+    private fun newUiAlbumEvent(album: Album){
+            val uiEvent = EventUtils.newUiAlbumEvent(album, UiEventType.PLAY_ALBUM)
```

Comment:
Same here on further encapsulating the event type - if `EventUtils.newUiAlbumEvent()` only handles `PLAY_ALBUM`, you shouldn't pass `PLAY_ALBUM` into the method. Is that the case here?

Reply:
I think for now the `EventUtils.newUiAlbumEvent()` handles only `PLAY_ALBUM` event, however, in the future, I think we would also be recording other UI_EVENTs like `ADD_ALBUM_TO_PLAYLIST, `PLAY_NEXT_ALBUM`, `ALBUM_SHUFFLE` etc if I'm not wrong

Reply:
Ok, then fine to leave as-is. 👍